src/Redux/Books/reducerBooks.js

```diff
@@ -0,0 +1,58 @@
+import axios from 'axios';
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+const BASE_URL = 'https://bobsburgers-api.herokuapp.com/characters/?limit=10&skip=489';
+const BASE_URL_SINGLE = 'https://bobsburgers-api.herokuapp.com/characters';
+const stateInit = [];
+const GET_DUME_BOOK = 'GET_DUME_BOOK';
+const GET_SINGLE_BOOK = 'GET_SINGLE_BOOK';
+
+const reducerDukeBooks = (state = stateInit, action) => {
+  switch (action.type) {
+    case `${GET_DUME_BOOK}/fulfilled`: return [...state, ...action.payload];
+    default: return state;
+  }
+};
+export const reducerSingleDukeBooks = (state = stateInit, action) => {
+  switch (action.type) {
+    case `${GET_SINGLE_BOOK}/fulfilled`: return { ...state, ...action.payload };
+    default: return state;
+  }
+};
+
+export const getDumeBook = createAsyncThunk(
```

Comment:
[OPTIONAL] I think it's a great idea to give names to your components and functions that are relative to the project theme. In your case, your API is about `Bob's burgers Characters`, and yet most of your components, actions, and function names have the keyword `books` instead of `characters`.